doc/lua_api.md

```diff
@@ -2387,6 +2387,48 @@ Table of resulting tool uses:
 
 
 
+Wear Bar Color
+==============
+
+'Wear Bar' is a property of items that defines the coloring
+of the bar that appears under damaged tools.
+If it is absent, the old behavior of green-yellow-red is
+used.
+
+Wear bar colors definition
+--------------------------
+
+### Syntax
+
+```lua
+{
+    -- 'constant' or 'linear'
+    -- (nil defaults to 'constant')
+    blend = "linear",
+    color_stops = {
+        [0.0] = "#ff0000",
+        [0.5] = "slateblue",
+        [1.0] = {r=0, g=255, b=0, a=150},
+    }
+}
+```
+
+### Blend mode `blend`
+
+* `linear`: blends smoothly between each defined color point.
+* `constant`: each color starts at its defined point, and continues up to the next point
+
+### Color stops `color_stops`
+
+Specified as `float` keys assigned to `ColorSpec` values.
```

Comment:
```suggestion
Specified as `ColorSpec` values assigned to `float` keys.
```

One thing that's odd about the API design: The keys in `color_stops` are not wear values, but durability values (`1 - wear`). This is not documented and inconsistent with the rest of the API. For consistency, it would make more sense to use wear values.